components/atomic/Tile/patterns/InvestigationTile/styles.js

```diff
@@ -63,52 +127,34 @@ export const Title = styled.div`
   }
 `;
 
-export const MixedLink = styled(BaseMixedLink)`
-  --Tile-bg-color: var(--turquoise85);
-  --Tile-color: var(--white);
-  --Tile-image-bg-color: var(--neutral20);
-
-  position: relative;
-  display: grid;
-  /* stylelint-disable declaration-block-no-redundant-longhand-properties */
-  grid-template-columns: 1fr;
-  grid-template-rows: ${fluidScale("234px", "164px")} 1fr;
-  grid-template-areas:
-    "image"
-    "title";
-  /* stylelint-enable declaration-block-no-redundant-longhand-properties */
-  align-content: start;
-  justify-items: center;
-  padding: 0;
-  border-radius: 16px;
-  color: var(--Tile-color);
-  text-decoration: none;
+export const EarlyAccessFlag = styled(EarlyAccess)`
+  position: absolute;
+  top: var(--Tile-image-margin-top);
+  right: 16px;
+  left: auto;
 
-  &[aria-disabled="true"] {
-    --Tile-bg-color: #6a6e6e;
-    --Tile-image-filter: grayscale(100%);
+  @media (max-width: ${BREAK_DESKTOP}) {
+    right: 0;
+    border-top-right-radius: 16px;
```

Comment:
Gave it a lil border radius (not in the original designs) so it tucked into the corner of the tile a bit more seamlessly.  Was okayed by Jose.